samples/languages/java/README.md

```diff
@@ -1,6 +1,16 @@
 # TensorFlow for Java: Examples
 
-Examples using the TensorFlow Java API.
+These examples include using pre-trained models for [image
+classification](label_image) and [object detection](object_detection),
+and driving the [training](training) of a pre-defined model - all using the
+TensorFlow Java APIs.
```

Comment:
Nit: Is it plural or not?  I think it's not.

Reply:
Made singular.